src/MidnightExample/Main.hs

```diff
@@ -1,4 +1,4 @@
-module MidnightExample.Main (module MidnightExample.Main, module MAlonzo.Code.MidnightExample.HSLedger) where
+module Main (module Main, module MAlonzo.Code.MidnightExample.HSLedger) where
```

Comment:
I removed the `MidnightExample` prefix from the module name in order to make it consistent with the main module from `Ledger` and make it easier to extract this in a function. 
We could also go the other way, and change `HSLedgerTest` module to `Ledger.HSLedgerTest` - I just chose one option arbitrarily, but happy to do the other one too